refinance/schemas/base.py

```diff
@@ -27,11 +27,11 @@ class BaseFilterSchema(BaseSchema):
     pass
 
 
-M = TypeVar("M")
+_M = TypeVar("_M", bound=BaseModel)
 
 
-class PaginationSchema(BaseSchema, Generic[M]):
-    items: List[M]
+class PaginationSchema(BaseSchema, Generic[_M]):
+    items: Sequence[_M]
```

Comment:
In case with pagination, `items` is always a List, not an abstract Sequence, AFAIK

Reply:
This is a broader type and is recommended for use in documentation.

https://docs.python.org/3/library/typing.html#typing.List

> Note that to annotate arguments, it is preferred to use an abstract collection type such as [Sequence](https://docs.python.org/3/library/typing.html#typing.Sequence) or [Iterable](https://docs.python.org/3/library/typing.html#typing.Iterable) rather than to use [list](https://docs.python.org/3/library/stdtypes.html#list) or typing.List.